lambdas/es/indexer/test/test_index.py

```diff
@@ -62,6 +70,30 @@
     },
     "userIdentity": {"principalId": "EXAMPLE"}
 }
+# typical response to head_object or get_object
+OBJECT_RESPONSE = {
+    'ResponseMetadata': ANY,
+    'AcceptRanges': 'bytes',
+    'LastModified': datetime.datetime(2019, 5, 30, 23, 27, 29, tzinfo=tzutc()),
```

Comment:
We might want to use `dateutil.tz.UTC` or maybe even `datetime.timezone.utc` instead of `tzutc()`.

Reply:
That string comes directly from the boto3 API, so I'm must matching what boto3 does.